api/envoy/config/filter/http/dynamic_forward_proxy/v2alpha/dynamic_forward_proxy.proto

```diff
@@ -37,7 +38,7 @@ message PerRouteConfig {
     // for host header forwarding which is not the desired outcome.
     //
     // [#next-major-version: host_rewrite_literal]
```

Comment:
nit remove comment: here and elsewhere if there are others.

Reply:
done